bullet/src/SimulationFeatures.hh

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (C) 2020 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
+#ifndef IGNITION_PHYSICS_BULLET_SRC_SIMULATIONFEATURES_HH_
+#define IGNITION_PHYSICS_BULLET_SRC_SIMULATIONFEATURES_HH_
+
+#include <vector>
+#include <ignition/physics/ForwardStep.hh>
+#include <ignition/physics/GetContacts.hh>
+
+#include "Base.hh"
+
+namespace ignition {
+namespace physics {
+namespace bullet {
+
+using SimulationFeatureList = FeatureList<
+  ForwardStep,
+  GetContactsFromLastStepFeature
+>;
+
+class SimulationFeatures :
+    public virtual Base,
+    public virtual Implements3d<SimulationFeatureList>
+{
+  public: void WorldForwardStep(
+      const Identity &_worldID,
+      ForwardStep::Output &_h,
+      ForwardStep::State &_x,
+      const ForwardStep::Input &_u) override;
+
+  public: std::vector<ContactInternal> GetContactsFromLastStep(
+      const Identity &/* _worldID */) const override
+      {
+        // TODO(lobotuerk): Implement contacts getter, could be like https://pybullet.org/Bullet/phpBB3/viewtopic.php?t=2855
+        ignerr << "Dummy GetContactsFromLastStep implementation";
```

Comment:
IIRC @Lobotuerk Added them to be able of adding collisions in `ign-gazebo`, the minimun set of features needed for collisions included this one [Collision features](https://github.com/ignitionrobotics/ign-gazebo/blob/d79aafd9de25c4d4b88bc210be3faefd40214387/src/systems/physics/Physics.cc#L303). So if this feature is not implemented, collisions are skipped.

Reply:
Okay. It's just that the whole idea of the `Feature` system in ign-physics is to avoid dummy implementations. If there's a plan to implement this in the short term, I'm good with merging this as is. If it's not going to be implemented, maybe we can look into making the `GetContactsFromLastStepFeature` feature an optional feature.

Reply:
https://github.com/ignitionrobotics/ign-gazebo/commit/a84ec51c2fddcc09eeb0ed26b31f8de669c1ce56 should resolve this